addons/actions/src/register.tsx

```diff
@@ -1,11 +1,27 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { addons, types } from '@storybook/addons';
+import { STORY_CHANGED } from '@storybook/core-events';
 import ActionLogger from './containers/ActionLogger';
-import { ADDON_ID, PANEL_ID, PARAM_KEY } from './constants';
+import { ADDON_ID, EVENT_ID, PANEL_ID, PARAM_KEY } from './constants';
 
 addons.register(ADDON_ID, (api) => {
   addons.addPanel(PANEL_ID, {
-    title: 'Actions',
+    title() {
+      const [actionsCount, setActionsCount] = useState(0);
+      const onEvent = () => setActionsCount(actionsCount + 1);
```

Comment:
I'd probably do something like:
```suggestion
      const onEvent = () => setActionsCount(previous => previous + 1);
```